recipes/makefile-project-workspace-creator/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+import os
+from conans import ConanFile, tools
+
+
+class MPCGeneratorConan(ConanFile):
+    name = "makefile-project-workspace-creator"
+    description = "The Makefile, Project and Workspace Creator"
+    license = "BSD-3-Clause"
+    homepage = "https://objectcomputing.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "makefile-project-workspace-creator", "objectcomputing", "installer")
+    settings = "os"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.settings.os == "Windows":
```

Comment:
```suggestion
        if tools.os_info.is_windows:
```

Reply:
@tarc Looking at it again, I think you were correct about using `self.settings.os`...
Sorry :see_no_evil: :smile: